shap/explainers/deep/deep_tf.py

```diff
@@ -146,8 +151,7 @@ def __init__(self, model, data, session=None, learning_phase_flags=None):
             else:
                 self.expected_value = tf.reduce_mean(self.model(self.data), 0)
 
-        if not tf.executing_eagerly():
-            self._init_between_tensors(self.model_output.op, self.model_inputs)
+        self._init_between_tensors(self.model_output.op, self.model_inputs)
```

Comment:
Did you try this in eager mode?

Reply:
Yes - thats why I included this change. If I do not run this code, I get the following error downstream: 
![image](https://user-images.githubusercontent.com/40830407/75446541-e19eb680-5967-11ea-87c2-a8ea09349716.png)